network-api/networkapi/fellows/templates/fellows_apply.html

```diff
@@ -10,27 +10,53 @@
+        <p>Mozilla Fellowships are a transformative experience for emerging leaders concerned with making the internet a safer, more accessible resource for everyone. Fellows expand their network and sphere of influence; design impactful projects with the potential to reach millions; and learn from and collaborate with a global community of thousands of Mozillians. Mozilla Fellows are also awarded competitive compensation and benefits. 
+        </p>
+        <h4 class="h4-light-black">Mozilla Fellows fit three particular profiles:</h4>
       </div>
     </div>
+    <div class="row">
+        <div class="col-12 col-md-3 d-flex justify-content-center align-items-center mb-3"> 
+          <img src="/_images/fellowships/svg/open_web-apply_now-optim.svg">
+        </div>
+        <div class="col-12 col-md-9">
+          <h3 class="h5-black mb-1">Open web activists</h3>
+          <p class="mb-4">Fellows who work in the realm of public interest technology, addressing issues like privacy, security, and inclusion online. These open web activists embed at leading human rights and civil society organizations around the world, lending their technical expertise.</p>
+        </div> 
+
+        <div class="col-12 col-md-3 d-flex justify-content-center align-items-center mb-3"> 
+          <img src="/_images/fellowships/svg/scientists-apply_now-optim.svg">
+        </div> 
+        <div class="col-12 col-md-9"> 
+          <h3 class="h5-black mb-1">Scientists and researchers</h3>
+          <p class="mb-4">Fellows who infuse open-source practices and principles into scientific research. “Science” is defined broadly; Fellows may work in the natural sciences, formal and applied sciences, or humanities, social sciences and library and information sciences. Fellows are based in the research institution with which they are currently affiliated.</p>
+        </div> 
+
+        <div class="col-12 col-md-3 d-flex justify-content-center align-items-center mb-3"> 
+          <img src="/_images/fellowships/svg/tech-policy-apply_now-optim.svg">
+        </div> 
+        <div class="col-12 col-md-9">
+          <h3 class="h5-black mb-1">Tech policy professionals</h3>
+          <p class="mb-4">Fellows who examine the interplay of technology and public policy, and craft legal, academic, and governmental solutions. These tech policy professionals are independent researchers and are not necessarily matched with a host organization or 
+          an institution.</p>
+        </div> 
+        
+        <div class="col-12 offset-md-3 mt-2 mb-5">
+          <a href="https://mozilla.fluxx.io/user_sessions/new" class="btn btn-normal">apply to be a fellow today</a>
+          <p class="small-gray mt-2">The call closes <strong>Friday, April 20, 2018 at 11:59PM PST</strong>.</p>
+        </div>
+    </div>
     <div class="row">
       <div class="col-12">
         {% include 'contact_banner.html' %}
       </div>
-    </div>
-  </div>
-</div>
+    </div> 
+  </div> 
+</div> 
```

Comment:
Can you remove the trailing space from these 3 lines? Also, not sure what editor you are using, but if it has a remove trailing spaces option that's best to leave turned on.

Reply:
I didn't even know this existed, Gavin! Thanks for the pro tip here :). Yay!